src/common/estimate_size/src/collections/btreemap.rs

```diff
@@ -62,20 +72,55 @@ where
     pub fn last_key_value(&self) -> Option<(&K, &V)> {
         self.inner.last_key_value()
     }
+}
+
+#[easy_ext::ext(BTreeMapAccessExt)]
```

Comment:
Why to use easy_ext here?

Reply:
Because these are not BTreeMap methods

Reply:
For me, extension trait is for
1. prevent overriding default impl
2. impl methods for 3rd party types (without using a newtype)

It's a little overkill to me here